packages/css/src/components/ListItem.css

```diff
@@ -0,0 +1,81 @@
+.ListItem {
+  padding: var(--space-l);
+  position: relative;
+  display: flex;
+  display: -webkit-flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
```

Comment:
Not needed, a `div` has this by default 😉

Reply:
👍